Common/Mpu6050.hpp

```diff
@@ -180,11 +178,22 @@ class Mpu6050 : public Accelerometer
     i2c_.WriteThenRead(kAccelerometerAddress,
                        { Value(RegisterMap::kDataConfig) }, &controlRegister,
                        1);
-    controlRegister = (controlRegister & 10111111) | (!is_active << 6);
+    auto sleepMask = bit::MaskFromRange(6);
+    // A value of 1 in bit 6 of the control register means the MPU6050 should be
+    // in sleep mode
+    if (is_active)
+    {
+      controlRegister = bit::Clear(controlRegister, sleepMask);
+    }
+    else
+    {
+      controlRegister = bit::Set(controlRegister, sleepMask);
+    }
```

Comment:
Rather than do a conditional here, you can simply insert the values. 
```suggestion
      // !is_active is required as the bit must be set to 0 in order to prevent it from sleeping.
      controlRegister = bit::Insert(controlRegister, !is_active, sleepMask);
```